### PR TITLE
156/sms notification

### PR DESCRIPTION
**Pull request by tazeen7**

### CODE REVIEW

File: test/In.ProjectEKA.HipServiceTest/SmsNotification/SmsNotificationControllerTest.cs [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+            gatewayClient.Setup(
+                    client =>
+                        client.SendDataToGateway(PATH_SMS_NOTIFY,
+                            gatewaySmsNotifyRequestRepresentation, cmSuffix, correlationId))
+                .Returns(Task.CompletedTask)
+                .Callback<string, GatewaySmsNotifyRepresentation, string, string>
+                ((path, gr, suffix, corId)
+                    => smsNotificationController.Accepted(onSmsContextRequest));
+        }
+    }
```
Comment (swatigogia2020) [inline test/In.ProjectEKA.HipServiceTest/SmsNotification/SmsNotificationControllerTest.cs:70]: add more test cases

**Review by swatigogia2020 — CHANGES_REQUESTED** [review]

File: src/In.ProjectEKA.HipService/SmsNotification/SmsNotificationController.cs [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+            //         if (!statusCodeResult.Result.StatusCode.Equals(StatusCodes.Status200OK))
+            //         {
+            //             return statusCodeResult.Result;
+            //         }
+            //     }
+            //     else
+            //     {
+            //         return StatusCode(StatusCodes.Status401Unauthorized);
+            //     }
+            // }
```
Comment (rohityawalkar04) [inline src/In.ProjectEKA.HipService/SmsNotification/SmsNotificationController.cs:69]: Please remove the comments if not needed

File: src/In.ProjectEKA.HipService/SmsNotification/SmsNotificationController.cs [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+            //         string sessionId = Request.Cookies[REPORTING_SESSION];
+            //
+            //         Task<StatusCodeResult> statusCodeResult = IsAuthorised(sessionId);
+            //         if (!statusCodeResult.Result.StatusCode.Equals(StatusCodes.Status200OK))
+            //         {
+            //             return statusCodeResult.Result;
+            //         }
+            //     }
+            //     else
+            //     {
```
Comment (rohit-yawalkar) [inline src/In.ProjectEKA.HipService/SmsNotification/SmsNotificationController.cs:66]: @tazeen7 Please remove the comments.

Comment (tazeen7) [reply src/In.ProjectEKA.HipService/SmsNotification/SmsNotificationController.cs:66]: Sure

**Review by rohit-yawalkar — APPROVED** [review]

**Review by swatigogia2020 — APPROVED** [review]

**Pull request merged**